platform/iphone/os_iphone.cpp

```diff
@@ -117,7 +157,8 @@ Error OSIPhone::initialize(const VideoMode &p_desired, int p_video_driver, int p
 	//visual_server->cursor_set_visible(false, 0);
 
 	// reset this to what it should be, it will have been set to 0 after visual_server->init() is called
-	RasterizerStorageGLES3::system_fbo = gl_view_base_fb;
+	if (use_gl3) RasterizerStorageGLES3::system_fbo = gl_view_base_fb;
```

Comment:
`clang-format` doesn't like this :)
https://travis-ci.org/godotengine/godot/jobs/466521381